### PR TITLE
Fixes #434 Add comparison tolerance

### LINKED ISSUE #434: Add comparison tolerance
Still some numeric comparisons are not using a comparison tolerance

### PR DESCRIPTION
**Pull request by Yuri05**

### CODE REVIEW

File: tests/testthat/helper-for-tests.R [hunk 1]
```diff
@@ -20,3 +20,7 @@ executeWithTestFile <- function(actionWithFile) {
   actionWithFile(newFile)
   file.remove(newFile)
 }
+
+comparisonTolerance <- function() {
+  1e-4
+}
```
Comment (Yuri05) [inline tests/testthat/helper-for-tests.R:26]: moved the definition of comparison tolerance here.

Comment (pchelle) [reply tests/testthat/helper-for-tests.R:26]: Good idea.
Would writing `comparisonTolerance <- function(tol = 1e-4) {tol}` be better ?
This way we can also tune to other values if necessary.

Comment (msevestre) [reply tests/testthat/helper-for-tests.R:26]: yes!

Comment (Yuri05) [reply tests/testthat/helper-for-tests.R:26]: done

**Review by pchelle — APPROVED** [review]

**Pull request merged**